server/sonar-db-dao/src/main/java/org/sonar/db/organization/OrganizationMemberDto.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.db.organization;
+
+public class OrganizationMemberDto {
+  private String uuid;
+  private String organizationUuid;
+  private Integer userId;
+
+  public String getUuid() {
+    return uuid;
+  }
+
+  public OrganizationMemberDto setUuid(String uuid) {
+    this.uuid = uuid;
```

Comment:
if you don't call `setUuid` the uuid is null so I'm not sure
- it will check data set by MyBatis but the DB garanties the non nullability
- it brings a false sense of safety
I'm just giving the contract: no @Nullable so it shouldn't be null and no @CheckForNull when in practice in can be null